packages/backend/src/routers/rating.ts

```diff
@@ -3,9 +3,11 @@ import { z } from "zod";
 import { TRPCError } from "@trpc/server";
 import { PendingRating, ratingBaseParser, RatingReport, reportParser } from "@backend/types/schema";
 import { DEPARTMENT_LIST } from "@backend/utils/const";
+import { getRateLimiter } from "@backend/middleware/rate-limiter";
 
 export const ratingsRouter = t.router({
     add: t.procedure
+        .use(getRateLimiter("ADD_RATING_LIMITER"))
         .input(
             ratingBaseParser.merge(
```

Comment:
Can we add a `.trim` to the `ratingBaseParser` have been finding some abuse where user just spams space to differentiate

Reply:
Added trims to various parse locations